src/coreclr/nativeaot/Runtime/thread.cpp

```diff
@@ -392,22 +392,13 @@ void Thread::Destroy()
 }
 
 #ifdef HOST_WASM
-<<<<<<< HEAD
-extern RtuObjectRef* GetShadowStackBottom();
-extern RtuObjectRef* GetShadowStackTop();
-=======
 extern OBJECTREF * t_pShadowStackTop;
```

Comment:
```suggestion
extern OBJECTREF* GetShadowStackBottom();
extern OBJECTREF* GetShadowStackTop();
```
These symbols are thread-local, so it is better to use the wrapper functions.